pkg/advisory/store.go

```diff
@@ -24,3 +24,7 @@ type Getter interface {
 	// valid, non-empty data.
 	Advisories(ctx context.Context, packageName string) ([]v2.PackageAdvisory, error)
 }
+
+type Putter interface {
```

Comment:
Are we also going to have a PitchingWedge and Driver at some point?

Reply:
Yes!